istio-1-3-1/cluster-local-gateway-1-3-1/base/pod-disruption-budget.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
```

Comment:
do we need the hpa also?  looks we have it for ingress gateway

Reply:
@yuzisun Interesting thought. The default Helm charts for Istio do not come with one. Should we add one like this?

```
apiVersion: autoscaling/v2beta1
kind: HorizontalPodAutoscaler
metadata:
  labels:
    app: cluster-local-gateway
    istio: cluster-local-gateway
  name: cluster-local-gateway
spec:
  maxReplicas: 5
  metrics:
  - resource:
      name: cpu
      targetAverageUtilization: 80
    type: Resource
  minReplicas: 1
  scaleTargetRef:
    apiVersion: apps/v1
    kind: Deployment
    name: cluster-local-gateway
```

Reply:
sgtm